### PR TITLE
[Fix]: added the darktheme support for Action Column in Alerts Listing Table.

### PR DESCRIPTION
**Pull request by Rajat-Dabade**
Screenshot 2023-10-31 at 10 12 12 AM

cc: @ankitnayan

### CODE REVIEW

File: frontend/src/components/DropDown/DropDown.tsx [hunk 1]
```diff
@@ -15,7 +18,7 @@ function DropDown({ element }: { element: JSX.Element[] }): JSX.Element {
 		<Dropdown menu={{ items }} className="Dropdown-container">
 			<Button
 				type="link"
-				className="Dropdown-button"
+				className={!isDardMode ? 'Dropdown-button--dark' : 'Dropdown-button'}
```
Comment (ankitnayan) [inline frontend/src/components/DropDown/DropDown.tsx:21]:
```suggestion
				className={!isDarkMode ? 'Dropdown-button--dark' : 'Dropdown-button'}
```

File: frontend/src/components/DropDown/DropDown.tsx [hunk 2]
```diff
@@ -2,8 +2,11 @@ import './DropDown.styles.scss';
 
 import { EllipsisOutlined } from '@ant-design/icons';
 import { Button, Dropdown, MenuProps, Space } from 'antd';
+import { useIsDarkMode } from 'hooks/useDarkMode';
 
 function DropDown({ element }: { element: JSX.Element[] }): JSX.Element {
+	const isDardMode = useIsDarkMode();
```
Comment (ankitnayan) [inline frontend/src/components/DropDown/DropDown.tsx:8]:
```suggestion
	const isDarkMode = useIsDarkMode();
```

**Review by ankitnayan — CHANGES_REQUESTED** [review]

File: frontend/src/components/DropDown/DropDown.tsx (hunk 1 above)
Comment (Rajat-Dabade) [reply frontend/src/components/DropDown/DropDown.tsx]: Done.

File: frontend/src/components/DropDown/DropDown.tsx (hunk 2 above)
Comment (Rajat-Dabade) [reply frontend/src/components/DropDown/DropDown.tsx]: Done

**Pull request merged**